cookies/views/giles.py

```diff
@@ -219,64 +219,107 @@ def test_giles_cleanup(request):
         pass
     return JsonResponse({'status': 'ok'})
 
+def _reupload_resources(request, form_data, filtered_objects):
+    upload_type = form_data.get('upload_type')
+    if upload_type == GilesLogForm.UPLOAD_ALL:
+        resources = filtered_objects.qs
+    else:
+        resources = form_data.get('resources')
+    auth_resources = auth.apply_filter(
+        ResourceAuthorization.EDIT,
+        request.user,
+        resources).filter(state__in=GilesUpload.ERROR_STATES)
+
+    context = {}
+    count_resources = resources.count()
+    count_success = 0
+    if auth_resources.count() > 0:
+        count_success = auth_resources.update(state=GilesUpload.PENDING)
+        context.update({
+            'reuploads_success': count_success,
+        })
+
+    if count_resources > 0:
+        context.update({
+            'reuploads_skipped': (count_resources - count_success),
+        })
+    return context
+
+def _change_priority(request, form_data, filtered_objects):
+    priority_level = form_data.get('priority')
+    if priority_level == GilesLogForm.PRIORITY_HIGH:
+        priority = GilesUpload.PRIORITY_HIGH
+    elif priority_level == GilesLogForm.PRIORITY_MEDIUM:
+        priority = GilesUpload.PRIORITY_MEDIUM
+    else:
+        priority = GilesUpload.PRIORITY_LOW
```

Comment:
Instead of these if/else statements to assign priority (same for the celery priorities), I would simply create a map and use that instead. Also, I'd use the same priority keys for the forms and the model. Then you don't have to map anything (only for the celery priority).